Library/Formula/glib-networking.rb

```diff
@@ -11,6 +23,8 @@ def pour_bottle?
     false
   end
 
+  # see https://github.com/Homebrew/homebrew/issues/36323
+  depends_on GnutlsWithoutP11Requirement
```

Comment:
AFAIK that syntax doesn't work. I thought we'd tried it before in places. Could be wrong though.